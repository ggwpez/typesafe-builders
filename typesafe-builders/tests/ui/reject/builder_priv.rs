mod other {
	use typesafe_builders::prelude::*;

	#[derive(Builder)]
	struct Struct {
		x: u8,
		y: u8,
	}
}

fn main() {
	let _ = other::Struct::builder();
}
