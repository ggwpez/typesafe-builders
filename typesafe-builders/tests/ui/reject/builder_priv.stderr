error[E0603]: struct `Struct` is private
  --> tests/ui/reject/builder_priv.rs:12:17
   |
12 |     let _ = other::Struct::builder();
   |                    ^^^^^^ private struct
   |
note: the struct `Struct` is defined here
  --> tests/ui/reject/builder_priv.rs:5:2
   |
5  |     struct Struct {
   |     ^^^^^^^^^^^^^
